api/services/contact_fields_mapping_service.py

```diff
@@ -16,18 +16,14 @@ def handle_contact_fields_data(self, jsonData):
             if not user_details:
                 return False
 
-            custom_fields = jsonData["contact"]["fields"]
-
-            for field_name, field_value in custom_fields.items():
-                if field_value:
-                    user_contact_field_mapping = (
-                        models.ContactFieldsMapping.query.get_by_field_name(
-                            field_name, field_value
-                        )
-                    )
-                    if user_contact_field_mapping:
+            contact_field_mappings = db.session.query(models.ContactFieldsMapping).all()
```

Comment:
Can we make a query function inside the ContactFieldsMapping function and call that?

Reply:
Done